frontend/storm_wiki/README.md

```diff
@@ -0,0 +1,125 @@
+# STORM wiki
+
+[STORM](https://github.com/stanford-oval/storm) frontend modified.
+
+## Features & Changes
+
+- dotenv for env vars
+- save result files as '*.md'
+- add date to to top of the result file
+- added STORM_TIMEZONE
+- configure dark and light theme colors
+- configure arize-phoenix in settings
+- configure searxng and other search engines in settings
+
+![storm](./assets/storm.png)
+
+<details><summary>screenshots</summary>
+
+article
+
+![storm-article](./assets/storm-article.png)
+
+creating new article with search and llm fallback options
+
+![new-article](./assets/new-article.png)
+
+add categories
+
+![category](./assets/category.png)
+
+configure search engines
+
+![configure-search](./assets/configure-search.png)
+
+dark and light themes
+
+![theme](./assets/theme.png)
+
+</details>
+
+## Prerequisites
+
+- Docker and Docker Compose
+- Required API keys (see main STORM repository)
+
+## Installation and Usage with Docker
+
+1. Clone the repository:
+   ```sh
+   git clone https://github.com/jaigouk/storm_wiki.git
+   cd storm_wiki
+   ```
+
+2. Set up environment variables:
+   ```sh
+   cp .env.example .env
+   # Edit .env with your preferred text editor to set the required variables
+   ```
+
+3. Set up Streamlit secrets:
+   ```sh
+   cp secrets.toml.example secrets.toml
```

Comment:
Shall it be copying to `.streamlit/secrets.toml`?

Reply:
yes. you are right.